docs/Targets.md

```diff
@@ -126,6 +126,24 @@ public var sampleData: Data {
 }
 ```
 
+With the `validationType` property, Moya also allows you to use Alamofire's requests validation.
```

Comment:
I think it would be helpful to
1. make a recommendation for what the default was (i.e. the upgrade path)
2. Link to the Alamofire docs on validationType

Reply:
1. Wouldn't it be better to put this in a migration_10_to_11.md instead?
2. Will do.